### DISCUSSION TITLE
Is there a way to re-use charting library?

### DISCUSSION THREAD

**Discussion by ceevee830** [mlflow/mlflow#7745] [Q&A] [answered]
We're interested in seeing if there is a way to reuse the charting library.  

REcharts is the full-featured REACT charting library we're considering at the moment.  But our backend team might use MLflow in the backend for data management.  There is an appeal to have MLflow charting on the front end along with MLflow in the backend.

Is there a way to re-use MLflow's charting library?

Thanks.
-Chris

**Comment by ceevee830** [comment]
Ping

**Comment by JHibbard** [comment, marked as answer]
Hi ceevee830,

MLflow is using a combination of plotting libraries: plotly.js, vega-lite.js, and leaflet for maps in the artifact viewer. If you'd like to make similar charts I'd suggest trying vega-lite.js and it's React.js embedding library or plotly.js.


Hope that helps!
Jim
